src/components/common/SearchBar/styles.ts

```diff
@@ -33,4 +35,11 @@ export default StyleSheet.create({
     borderRadius: 16,
     marginLeft: 8,
   },
+  inputStyle: {
+    ...fontMaker({
+      size: 16,
+      weight: SEMIBOLD,
+      color: Colors.White.Pure,
+    }),
+  },
```

Comment:
This should work right?
```suggestion
  inputStyle: fontMaker({
      size: 16,
      weight: SEMIBOLD,
      color: Colors.White.Pure,
    }),
```

Reply:
yeaah